ts/src/bitfinex2.ts

```diff
@@ -778,19 +779,36 @@ export default class bitfinex2 extends Exchange {
         const networksById = {
             'BITCOIN': 'BTC',
             'LITECOIN': 'LTC',
-            'ETHEREUM': 'ERC20',
-            'TETHERUSE': 'ERC20',
+            'ERC20': 'ETH',
+            'TETHERUSE': 'ETH',
             'TETHERUSO': 'OMNI',
             'TETHERUSL': 'LIQUID',
-            'TETHERUSX': 'TRC20',
+            'TETHERUSX': 'TRX',
             'TETHERUSS': 'EOS',
             'TETHERUSDTAVAX': 'AVAX',
             'TETHERUSDTSOL': 'SOL',
-            'TETHERUSDTALG': 'ALGO',
+            'TETHERUSDTALG': 'ALG',
             'TETHERUSDTBCH': 'BCH',
             'TETHERUSDTKSM': 'KSM',
             'TETHERUSDTDVF': 'DVF',
             'TETHERUSDTOMG': 'OMG',
+            'TETHERUSDTXTZ': 'XTZ',
+            'TETHERUSDTNEAR': 'NEAR',
+            'TETHERUSDTDOT': 'DOT',
+            'TETHERUSDTPLY': 'PLY',
+            'TETHERUSDTKAVA': 'KAVA',
+            'TETHERMXNTE': 'ETH',
+            'TETHERXAUTE': 'ETH',
+            'TETHEREUE': 'ETH',
+            'TETHERCNHTX': 'ETH',
+            'TETHERCNHTE': 'ETH',
+            'TETHERUSDTZK': 'ZK2',
+            'XAUT': 'ETH',
+            'CNHT': 'ETH',
+            'EUT': 'BSC',
+            'MXNT': 'ETH',
+            'NEOGAS': 'NEO',
+            'MATICM': 'NEO',
```

Comment:
I think it might be an error

Reply:
Yeah it was probably a wrong copy pasta as `MATICM` is mapped to polygon, sorry